test/functional/testrunner.js

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/* eslint-disable func-style */
+/* eslint-disable no-console */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+/* eslint-disable no-useless-concat */
```

Comment:
That's a lot of lint disabling. If the rules are disabled because you just don't like the rules, let's bring it up with the team and make that decision repo-wide. If rules are disabled because this file should be an exception (disabling no-console seems a likely candidate), they're probably fine to keep.